assets/aml-benchmark/scripts/custom_dataset_preprocessors/base_preprocessor_template.py

```diff
@@ -20,15 +20,12 @@ def _parse_args():
     return argss
 
 
-def _read_jsonl_file(file_path: str) -> List[Dict[str, Any]]:
-    """Read `.jsonl` file and return a list of dictionaries.
+def _read_input_file(file_path: str) -> List[Dict[str, Any]]:
```

Comment:
May I know why does this function even exist? We should try to use the one that we have in `io.py`.